MeshLib/Elements/PyramidRule5.h

```diff
@@ -80,12 +80,11 @@ class PyramidRule5 : public CellRule
     static const Element* getFace(const Element* e, unsigned i);
 
     /**
-     * Checks if a point is inside the element.
-     * @param pnt a 3D MathLib::Point3d object
-     * @param eps tolerance for numerical algorithm used or computing the property
-     * @return true if the point is not outside the element, false otherwise
+     * \copydoc MeshLib::Element::isPntInElement()
+     * @param nodes the nodes of the element.
```

Comment:
Yes. Other parameters docs are copied (probably must have exactly the same types and names, which they do). Only extra parameters needs documentation.